SU-SWS/saa_victoria_subtheme
# Adapt 476  sidebar nav

# READY FOR REVIEW

# Summary
- Adjust side nav colors to be like the top main nav. 

# Review By (Date)
- 7/21

# Urgency
- needed today

# Steps to Test

1. pull down
1. clear cache
2. see side nav link colors like this:

- Regular links are black
- Current page has the saa vivid red E50808
- And all links when hovered over cardinal red #8C1515

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket testing for release
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
